### PR TITLE
Update pipeline.yml to verify changeset in PR build

### PR DESCRIPTION
**Pull request by IainSAP**
Adds verification of change sets as a broken change set will be otherwise merged to main and block all subsequent publishing.

### CODE REVIEW

**Comment by IainSAP** [comment]
PR build will be fixed with https://github.com/SAP/open-ux-tools/pull/2655

File: .changeset/config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "commit": false,
     "linked": [],
     "access": "public",
-    "baseBranch": "main",
+    "baseBranch": "origin/main",
```
Comment (IainSAP) [inline .changeset/config.json:7]: This fixes multiple issues. 1) Locally where there are multiple remotes referring to `main` (e.g. forks) 2) Some tools require the fully qualified branch name

**Review by devinea — APPROVED** [review]
Nice update

**Review by longieirl — APPROVED** [review]
Great!

**Pull request merged**